packages/runtime.win10-arm64.native.Microsoft.SqlToolsService/runtime.win10-arm64.native.Microsoft.SqlToolsService.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+	<TargetFramework>net5.0</TargetFramework>
```

Comment:
it would be nice to consolidate the framework version and perhaps other common build properties in a shared build properties file.  We bump the framework basically everything a new one comes out so have these values hardcoded  makes that process a little more difficult.

Reply:
+1, for both global common properties like the framework version and then common properties for the packages (PackageReleaseNotes, PackageVersion, etc)

Reply:
I agree, I figure you guys understand how you want to do that for your project, and will make these projects align with your build.